src/bp/core/app.ts

```diff
@@ -12,30 +12,35 @@ import Database from './database'
 import { GhostService } from './services'
 import { TYPES } from './types'
 
-let botpress
-let logger: LoggerProvider | undefined
-let config: ConfigProvider | undefined
-let ghost: GhostService | undefined
-let database: Database | undefined
-let localActionServer: LocalActionServerImpl | undefined
+export interface BotpressApp {
+  botpress: Core
+  logger: LoggerProvider
+  config: ConfigProvider
+  ghost: GhostService
+  database: Database
+  localActionServer: LocalActionServerImpl
+}
+
+export function createApp(): BotpressApp {
+  try {
+    const app = {
+      botpress: container.get<Core>(TYPES.Botpress),
+      logger: container.get<LoggerProvider>(TYPES.LoggerProvider),
+      config: container.get<ConfigProvider>(TYPES.ConfigProvider),
+      ghost: container.get<GhostService>(TYPES.GhostService),
+      database: container.get<Database>(TYPES.Database),
+      localActionServer: container.get<LocalActionServerImpl>(TYPES.LocalActionServer),
+      licensing: container.get<LicensingService>(TYPES.LicensingService)
+    }
```

Comment:
I made this function otherwise the server would be initialized by just including the file somewhere.